assemblies/karaf-features/src/main/feature/feature.xml

```diff
@@ -469,6 +471,7 @@
     <bundle start-level="82">mvn:org.opencastproject/opencast-oaipmh/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-oaipmh-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-oaipmh-persistence/${project.version}</bundle>
+    <bundle start-level="82">mvn:org.opencastproject/opencast-playlists/${project.version}</bundle>
```

Comment:
I've only added it to "admin" to avoid cluttering other distributions with a service that will never see any use there (e.g. why would this ever be used on a worker?). If people want it on other distributions I don't mind adding those (presentation seems reasonable?).